### PR TITLE
Polygon integration

### PR DESCRIPTION
**Pull request by Rakeally**

### CODE REVIEW

File: ios/Classes/WalletHandler.swift [hunk 1]
```diff
@@ -18,9 +18,9 @@ class WalletHandler {
         "NEAR"  : NEAR(),
         "XLM"   : XLM(),
         "BNB"   : BNB(),
-        "BSC"   : BSC(),
+        "BSC"   : ETH(),
```
Comment (baahkusi) [inline ios/Classes/WalletHandler.swift:21]: Let's remove this then and just always use ETH . So all ETH compatible will just use the ETH

File: ios/Classes/WalletHandler.swift [hunk 2]
```diff
@@ -18,9 +18,9 @@ class WalletHandler {
...
         "DOGE"  : DOGE(),
-        "MATIC" : MATIC(),
+        "MATIC" : ETH()
```
Comment (baahkusi) [inline ios/Classes/WalletHandler.swift:23]: Let's remove this then and just always use ETH . So all ETH compatible will just use the ETH

File: android/src/main/kotlin/africa/ejara/trustdart/WalletHandler.kt [hunk 3]
```diff
@@ -29,13 +29,13 @@ class WalletHandler {
             "NEAR"  to NEAR(),
             "XLM"   to XLM(),
             "BNB"   to BNB(),
-            "BSC"   to BSC(),
+            "BSC"   to ETH(),
```
Comment (baahkusi) [inline android/src/main/kotlin/africa/ejara/trustdart/WalletHandler.kt:32]: Let's remove this then and just always use ETH . So all ETH compatible will just use the ETH

File: android/src/main/kotlin/africa/ejara/trustdart/WalletHandler.kt [hunk 4]
```diff
@@ -29,13 +29,13 @@ class WalletHandler {
...
             "DOGE"  to DOGE(),
-            "MATIC" to MATIC(),
+            "MATIC" to ETH(),
```
Comment (baahkusi) [inline android/src/main/kotlin/africa/ejara/trustdart/WalletHandler.kt:34]: Let's remove this then and just always use ETH . So all ETH compatible will just use the ETH

**Review by baahkusi — CHANGES_REQUESTED** [review]

**Review by jayluxferro — APPROVED** [review]

**Review by baahkusi — APPROVED** [review]

**Pull request merged**